core/store/models/job_run.go

```diff
@@ -239,8 +241,8 @@ func NewRunRequest(requestParams JSON) *RunRequest {
 // TaskRun stores the Task and represents the status of the
 // Task to be ran.
 type TaskRun struct {
-	ID                               *ID           `json:"id" gorm:"primary_key;not null"`
-	JobRunID                         *ID           `json:"-"`
+	ID                               *ID           `json:"id" gorm:"type:uuid;primary_key;not null"`
+	JobRunID                         *ID           `json:"-" gorm:"type:uuid"`
```

Comment:
Would be good to use either `uuid` type or `null.UUID` if it is nullable, and remove this `*ID` thing completely

Reply:
yeah I think out of scope for this PR though as its large already